package.json

```diff
@@ -3,7 +3,7 @@
   "version": "2.36.48",
   "private": true,
   "scripts": {
-    "build-storybook": "build-storybook",
+    "build-storybook": "NODE_OPTIONS=--max-old-space-size=6144 build-storybook",
```

Comment:
I think this makes sense for now!!! I wonder if it will magically fix itself after we update everything because parsley is on the same version of SideNav and node with no special configurations. 

if not maybe let's try updating other libraries like `vite` and `@storybook/builder-vite` after all of the LG components have been updated!

Reply:
Yes I'm inclined to think all this will help! I did run [a patch](https://spruce.mongodb.com/version/63615bf59ccd4e772e87671a/tasks?sorts=STATUS%3AASC%3BBASE_STATUS%3ADESC) on Vite 3/updated storybook builder that sadly failed in the same way — but I didn't go all the way in terms of upgrading our Storybook libraries. I think once Storybook v7 is released with first-class Vite support we may see better results.